javaspec-runner/src/test/java/info/javaspec/spec/FieldSpecTest.java

```diff
@@ -18,88 +17,46 @@
 import org.mockito.Mockito;
 
 import java.lang.reflect.Field;
-import java.lang.reflect.InvocationTargetException;
 import java.util.LinkedList;
 import java.util.List;
 
 import static com.google.common.collect.Lists.newArrayList;
-import static info.javaspec.testutil.Assertions.capture;
+import static info.javaspec.junit.Descriptions.isTestDescription;
 import static info.javaspec.testutil.Matchers.isThrowableMatching;
-import static info.javaspec.testutil.Matchers.matchesRegex;
+import static java.util.stream.Collectors.toList;
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.contains;
 import static org.hamcrest.Matchers.*;
-import static org.junit.Assert.assertEquals;
 import static org.junit.runner.Description.createSuiteDescription;
 import static org.mockito.Mockito.*;
 
 @RunWith(HierarchicalContextRunner.class)
-public class SpecTest {
+public class FieldSpecTest {
   private Spec subject;
   private final List<String> events = new LinkedList<>();
 
-  public class run {
-    private final RunNotifier notifier = mock(RunNotifier.class);
+  public class addDescriptionTo {
+    public class givenASuite {
+      private Description suite;
 
-    @Test
-    public void itThrowsFaultyClassInitializer_givenAnExplodingClassInitializer() throws Exception {
-      FaultyClassInitializer ex = capture(FaultyClassInitializer.class, () -> {
```

Comment:
Incorrect behavior (but at least I could demonstrate that it was incorrect).  Specs aren't supposed to throw anything up to the test runner.